lib/filter-stream.js

```diff
@@ -24,6 +24,7 @@ function FilterStream( blockMap, options ) {
   /** @type {Object} options */
   this.options = options || {}
 
+  this.options.readableObjectMode = true
```

Comment:
This is only specifically overridden to prevent the readable state buffer from concatenating the output blocks (dropping the block's position & address properties), but otherwise passing the options straight into the super ctor is intended, as that allows for control of things like event emitter and other stream options that wouldn't make sense to specifically map here (i.e. writableObjectMode, maxListeners, highWaterMark, ...)